### PR TITLE
OP-1368 | Make Inventory status language independent

### PR DESCRIPTION
**Pull request by ArnaudFofou**
See OP-1368.

### CODE REVIEW

File: src/main/java/org/isf/medicalinventory/manager/MedicalInventoryManager.java [hunk 1]
```diff
@@ -71,6 +74,8 @@ public class MedicalInventoryManager {
 	private SupplierBrowserManager supplierManager;
 
 	private WardBrowserManager wardManager;
+	
+	protected HashMap<String, String> statusHashMap;
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/medicalinventory/manager/MedicalInventoryManager.java]:
```suggestion
	protected Map<String, String> statusHashMap;
```

**Review by mwithi — DISMISSED** [review]
LGTM. Thanks a lot!

**Review by dbmalkovsky — DISMISSED** [review]
I second @mwithi LGTM

**Comment by dbmalkovsky** [comment]
@ArnaudFofou please resolved the conflicting files; thanks.

**Comment by mwithi** [comment]
> @ArnaudFofou please resolved the conflicting files; thanks.

Sorry, I had to resolve them myself because I had already merged GUI when I realized there were conflicts here.

**Pull request merged**